src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -182,6 +182,16 @@ impl HtmlHandlebars {
         handlebars.register_helper("previous", Box::new(helpers::navigation::previous));
         handlebars.register_helper("next", Box::new(helpers::navigation::next));
     }
+
+    fn copy_additional_css(&self, book: &MDBook) -> Result<(), Box<Error>> {
```

Comment:
Hi just minor suggestion. This function copies also additional js files so maybe rename it.